eng/common/scripts/Package-Properties.ps1

```diff
@@ -219,12 +228,13 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
             }
             $filePath = (Join-Path $RepoRoot $file)
 
+            # handle direct changes to packages
             $shouldInclude = $filePath -like (Join-Path "$pkgDirectory" "*")
 
-            # this implementation guesses the working directory of the ci.yml
+            # handle changes to files that are RELATED to each package
             foreach($triggerPath in $triggeringPaths) {
                 $resolvedRelativePath = (Join-Path $RepoRoot $triggerPath)
-                # utilize the various trigger paths against the targeted file here
+
                 if (!$triggerPath.StartsWith("/")){
                     $resolvedRelativePath = (Join-Path $RepoRoot "sdk" "$($pkg.ServiceDirectory)" $triggerPath)
```

Comment:
We can utilize the `triggeringPaths` details here to grab the actual containing directory of the `ci.yml`....that may be FUD though and $pkg.ServiceDirectory is entirely reliable even for deeper ci.yml files.